src/zzz_od/application/hollow_zero/lost_void/operation/update_priority_operation.py

```
@@ -89,17 +116,16 @@ def recognize_and_store(self) -> OperationRoundResult:
-                # 4. 匹配并提取category
-                artifact = self.ctx.lost_void.match_artifact_by_ocr_full(best_candidate.text)
-                # 5. 判断是否为“武备”
-                if artifact and artifact.is_gear:
-                    log.info(f"发现低等级【武备】: {artifact.display_name}，添加优先级: {artifact.category}")
-                    new_priorities.add(artifact.category)
-                elif artifact:
-                    log.debug(f"发现低等级【非武备】藏品: {artifact.display_name}，已忽略")
+                # 4. 直接按 [a:b] 中的 a 提取动态优先级分类
+                priority_category = self._extract_priority_category_from_text(best_candidate.text)
+                if priority_category is not None:
+                    log.info(f'发现低等级藏品: {best_candidate.text}，添加优先级: {priority_category}')
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)